src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssKeysAndValues = sourceString
+    .split(';')
+    .filter((item) => item.trim().length)
+    .reduce((prev, item) => {
+      const index = item.indexOf(':');
+
+      return {
+        ...prev,
+        [item.slice(0, index).trim()]: item.slice(index + 1).trim(),
```

Comment:
The chained method calls should have the same indentation to improve readability. Align `.filter` and `.reduce` with `.split` by placing them on new lines with the same level of indentation. [CODE STYLE]